starknet_py/net/udc_deployer/deployer.py

```diff
@@ -13,14 +13,12 @@
 from starknet_py.serialization import serializer_for_function
 from starknet_py.utils.contructor_args_translator import translate_constructor_args
 from starknet_py.utils.crypto.facade import pedersen_hash
-from starknet_py.utils.sync import add_sync_methods
 
 ContractDeployment = NamedTuple(
     "ContractDeployment", [("call", Call), ("address", int)]
 )
 
 
-@add_sync_methods
```

Comment:
Isn't this breaking?

Reply:
Nope, because there isn't any `async` method in the Deployer 😅